### ISSUE TITLE
SoVITS训练卡在这步不动Detected call of `lr_scheduler.step()` before `optimizer.step()`

### ISSUE THREAD

**Issue by singsongs7731** [RVC-Boss/GPT-SoVITS#857]
wav_data_len: 99
100%|██████████████████████████████████████████████████████████████████████████████████████████| 99/99 [00:00<?, ?it/s]
skipped_phone:  0 , skipped_dur:  0
total left:  99
F:\GPT-SoVITS-main\runtime\lib\site-packages\torch\nn\utils\weight_norm.py:28: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
ssl_proj.weight not requires_grad
ssl_proj.bias not requires_grad
F:\GPT-SoVITS-main\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
INFO:ym33:loaded pretrained GPT_SoVITS/pretrained_models/s2G488k.pth
<All keys matched successfully>
INFO:ym33:loaded pretrained GPT_SoVITS/pretrained_models/s2D488k.pth
<All keys matched successfully>
F:\GPT-SoVITS-main\runtime\lib\site-packages\torch\optim\lr_scheduler.py:143: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "

**Comment by singsongs7731** [comment]
解决了吗

**Comment by XXXXRT666** [comment]
可以尝试一下关闭共享显存

**Comment by MeAquaaa** [comment]
同样的问题

**Comment by 2268190072** [comment]
有爸爸知道怎么解决吗

**Comment by YuGhi-bug** [comment]
相同問題

**Comment by YuGhi-bug** [comment]
> 开大虚拟内存

好像解決了 設定成系統自動設定

**Comment by Hunt3rseeker** [comment]
This is still an issue for me.
